keps/sig-apps/3329-retriable-and-non-retriable-failures/README.md

```diff
@@ -878,6 +901,12 @@ As a solution we implement a worker, part of the disruption
 controller, which clears the pod condition added if `DeletionTimestamp` is
 not added to the pod for a long enough time (for example 2 minutes).
 
+#### Waiting for pods to be marked as Failed
+
+We discovered that the Job controller will start new pods as soon as they are marked for deletion. [Jobs create replace pods as soon as Pod is marked for deletion](https://github.com/kubernetes/kubernetes/issues/115844)).
+
+This is being treated as a bug but we have to enhance the API to allow this.  We are adding `DelayPodRecreationUntilTerminal` to the PodFailurePolicy spec.
```

Comment:
@mimowo has suggested a new field name: https://github.com/kubernetes/kubernetes/pull/117015#discussion_r1156004718

With the current implementation of this, we found that we can include terminating pods as active.  

Should we change `DelayPodRecreationUntilTerminal` to be `TerminatingAsActive`?

Reply:
+1 for `activeUntilTerminal`, I also prefer it slightly over the previous option.